hibp.js

```diff
@@ -23,17 +23,37 @@ const HIBP = {
     return Object.assign(options, hibpOptions);
   },
 
+  async _throttledGot (url, reqOptions, tryCount = 1) {
+    try {
+      return await got(url, reqOptions);
+    } catch (err) {
+      console.error("got an error: " + err);
+      if (err.statusCode === 429) {
+        console.log("got a 429, tryCount: ", tryCount);
+        if (tryCount >= AppConstants.HIBP_THROTTLE_MAX_TRIES) {
+          throw new Error(err.message);
```

Comment:
Should we throw a _new_ error, or just rethrow the existing error?

```js
throw err;
```

Not sure if there is anything useful in the original stack or not. 🤷‍♂️

Reply:
Good idea. Changed to `console.error` the full stack, but throw a more user-friendly "Too many connections to HIBP." error to calling code.